Bugzilla/Quantum.pm

```diff
@@ -12,6 +12,7 @@ use Mojo::Base 'Mojolicious';
 use CGI::Compile;
 use utf8;
 use Encode;
+use FileHandle;    # this is for compat back to 5.10
```

Comment:
This wasn't related to this commit, except without it the vagrant vm doesn't work for me.